test/unit/gateways/eway_managed_test.rb

```diff
@@ -25,11 +25,11 @@ def setup
                   :title => 'Mr.',
                   :phone => '(555)555-5555'
                },
-               :email => 'someguy1232@fakeemail.net',
-               :order_id => '1000',
-               :customer => 'mycustomerref',
-               :description => 'My Description',
-               :invoice => 'invoice-4567'
+                 :email => 'someguy1232@fakeemail.net',
```

Comment:
I'm confused again what's going on here. Seems like these should align with the `}` on 27?

Reply:
This rubocop is trying to align the `:`'s with each other. Not he to the `{`